e2e/cypress/integration/multi_team_and_dm/close_gm_via_menu_spec.js

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @multi_team_and_dm
+
+describe('Multi-user group messages', () => {
+    let testUser;
+    let testTeam;
+    before(() => {
+        // # Create a new team
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam = team;
+
+            // # Add 10 users to the team
+            Cypress._.times(3, () => createUserAndAddToTeam(testTeam));
+        });
+    });
+
+    it('MM-T1799 Should Close Group Message from channel menu', () => {
+        // # Login as test user
+        cy.apiLogin(testUser);
+
+        // # Go to town-square channel
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the 'Direct messages' dialog to create a new direct message
+        cy.get('#addDirectChannel').click();
```

Comment:
I'm getting intermittent failing result here. I'd suggest to add timeout since it's right after visiting a channel where loading a page sometimes takes time.

```javascript
cy.get('#addDirectChannel', {timeout: TIMEOUTS.HALF_MIN}).should('be.visible').click();
```